### PR TITLE
{tools}[GCC/12.3.0] ega-quickview v20240620, FUSE v3.16.2, crypt4gh v1.7

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/c/crypt4gh/crypt4gh-1.7-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+use_pip = True
+
+exts_list = [
+    ('PyNaCl', '1.5.0', {
+        'modulename': 'nacl',
+        'checksums': ['8ac7448f09ab85811607bdd21ec2464495ac8b7c66d146bf545b0f08fb9220ba'],
+    }),
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/c/crypt4gh/crypt4gh-1.7-GCC-12.3.0.eb:28]: Both of these, in the exact same versions, are provided by `Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb`.

File: easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+source_urls = [GITHUB_SOURCE]
+sources = [{
+    "download_filename": "%s.tar.gz" % local_commit,
+    "filename": "%(name)s-%(version)s.tar.gz",
+}]
+checksums = ['90836e42009736a8e20a2569918638f3cb2b53574265ca4f4bed7abd81a5e887']
+
+builddependencies = [
+    ('make', '4.4.1'),
+    ('Autoconf', '2.71'),
```
Comment (branfosj) [inline easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb:22]:
```suggestion
    ('Autotools', '20220317'),
```

```
Can't exec "aclocal": No such file or directory at /project/def-maintainers/boegelbot/rocky9/zen3/software/Autoconf/2.71-GCCcore-12.3.0/share/autoconf/Autom4te/FileUtils.pm line 274.
autoreconf: error: aclocal failed with exit status: 2
```
`aclocal` is provided by `Automake` - so we should provide all of the `Autotools` here.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0105u03b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/56cf19c843b46e502056fdc8b5d58bd5 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @pavelToman!

File: easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'ega-quickview'
```
Comment (boegel) [inline easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb:3]: I would prefer using `EGA-QuickView` as software name (see also description)

fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/20888

**Pull request merged**